TAF/testScenarios/functionalTest/V2-API/app-service/info/GET.robot

```diff
@@ -1,28 +1,37 @@
 *** Settings ***
-Resource         TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource  TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource  TAF/testCaseModules/keywords/app-service/AppServiceAPI.robot
 Suite Setup      Run Keywords  Setup Suite  AND  Deploy App Service
+...                            AND  Set Suite Variable  ${url}  ${AppServiceUrl_blackbox}
+...                            AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
 Suite Teardown   Run Keywords  Suite Teardown  AND  Remove App Service
+...                            AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Remove Token
 
 *** Variables ***
 ${SUITE}          App-Service GET Testcases
 ${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/app-service-get.log
 ${edgex_profile}  blackbox-tests
+${AppServiceUrl_blackbox}  http://${BASE_URL}:48095
+${api_version}  v2
 
 *** Test Cases ***
```

Comment:
All response V2 DTOs now have apiVersion. Should be verifying that it is present?

Reply:
Added apiVersion check.